reporting/api/src/main/java/org/jboss/windup/reporting/model/JarDependenciesReportModel.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.windup.reporting.model;
+
+import org.jboss.windup.graph.model.ProjectModel;
+
+import com.tinkerpop.blueprints.Direction;
+import com.tinkerpop.frames.Incidence;
+import com.tinkerpop.frames.modules.typedgraph.TypeValue;
+
+/**
+ * This represents Jar Dependencies report model
+ * 
+ * @author mnovotny
+ *
+ */
+@TypeValue(JarDependenciesReportModel.TYPE)
+public interface JarDependenciesReportModel extends ApplicationReportModel
+{
+    String TYPE = "jarDependenciesReport";
+
+    String JAR_REPORT_TO_PROJECT_MODEL = "jarDepsToProjectModel"; 
+
+    /**
+     * Contains all {@link ProjectModel}s that contain this file
+     */
+    @Incidence(label = JAR_REPORT_TO_PROJECT_MODEL, direction = Direction.OUT)
```

Comment:
The traversal can give you accurate path information based upon the application being traversed. It is basically impossible to get a full path back to root without one at this point.

I was thinking that there should be a link that looks something like this:

ReportModel -> Dependency (basically this groups archives by SHA1) -> Edge (with filename information attached to this edge) -> ArchiveModel